Source/gui/SNTNotificationManager.m

```diff
@@ -129,6 +142,14 @@ - (void)queueMessage:(SNTMessageWindowController *)pendingMsg {
   }
 }
 
+- (void)postDistributedNotificationWithUserInfo:(NSDictionary *)userInfo notificationName:(NSString *)notificationName {
```

Comment:
We enforce a per-line character limit of 100 characters. Break this line before `notificationName:` and then line up the colons of each parameter.

Reply:
fixed with `./Testing/fix.sh`